Content.Server/Salvage/SalvageMagnetComponent.cs

```diff
@@ -94,8 +94,7 @@ public sealed class SalvageMagnetComponent : SharedSalvageMagnetComponent
         public int PreviousCharge = 5;
 
     }
-
-    [CopyByRef, DataRecord]
+    [CopyByRef, DataRecord, Serializable, DataDefinition] // TODO PSS: Check if this is actually being properly saved/loaded.
```

Comment:
Afaik this shouldn't need serializable or datadefinition, `DataRecord` should be enough.
It was missing before and only recently added in #14715, which might've caused errors before.

Reply:
With DataRecord by itself it was throwing an error for a specific map that has a SalvageMagnet